AppCenterCrashes/AppCenterCrashes/Internals/Util/MSACErrorLogFormatter.m

```diff
@@ -736,6 +734,39 @@ + (MSACBinaryImageType)imageTypeForImagePath:(NSString *)imagePath processPath:(
 
 #pragma mark - Helpers
 
++ (MSACDevice *)deviceForTimestamp:(NSDate *)timestamp crashReport:(PLCrashReport *)report {
+  MSACDevice *device = [[MSACDeviceTracker sharedInstance] deviceForTimestamp:timestamp];
+  MSACDevice *alteredDevice = [MSACDevice new];
+
+  // Merge PLCR system information with our device information as the PLCR report
+  // is more relevant in cases when the time on the device has been manually changed.
+  alteredDevice.osVersion = report.systemInfo.operatingSystemVersion;
```

Comment:
Shouldn't we fallback to the value from device tracker if crash reporter's field is empty or nil?